experimental/PlaneCurve/ProjEllipticCurve.jl

```diff
@@ -174,7 +174,7 @@ mutable struct ProjEllipticCurve{S} <: ProjectivePlaneCurve{S}
   degree::Int
   components::Dict{ProjEllipticCurve{S}, Int}
   point::Oscar.Geometry.ProjSpcElem{S}
-  maps::Vector{Oscar.AlgHom{S}}
+  maps
```

Comment:
Can we get at least a minimal type annotation? E.g. even this would be helpful:
```suggestion
  maps::Vector
```

Reply:
Helpful for what exactly? Documentation?